content/content-strategy/understand-content-ecosystem/cards.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: cards/cards
 cards:
-  - headline: 'Content discovery'
-    text: Start by listing all of the content channels used by the agency. 
+  - headline: 'Discover content channels'
+    text: Start by listing all of the content channels used by the agency.
     link: '/content-strategy/understand-channel-ecosystem/content-discovery/'
```

Comment:
@annekemyers broken link: needs to point to /understand-content-ecosystem/.. etc

Reply:
(I've fixed this now in a separate commit)